lib/themes/dosomething/paraneue_dosomething/js/components/Button.js

```diff
@@ -0,0 +1,37 @@
+const React = require('react');
+
+/**
+ * Button Component
+ * <Button />
+ */
+const Button = (props) => {
+    const classes = ['button'].concat(props.classes);
```

Comment:
I'd recommend removing `props.classes`, since it breaks encapsulation of the component's styles (and there probably shouldn't be any need to pass arbitrary classes in that wouldn't be better served as part of the button).